identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -60,8 +60,7 @@ trait ConsentsJourney
 
               case Right(updatedUser) =>
                 logger.info(s"Successfully set hasRepermissioned flag for user ${request.user.id}")
-                val page = IdentityPage("/complete-consents", "Complete Consents", isFlow = true)
-                consentCompleteView(page, returnUrl)
+                Future.successful(SeeOther(s"${routes.EditProfileController.displayConsentComplete().url}?returnUrl=${returnUrl}"))
```

Comment:
I think both the `Right` and `Left` could now avoid returning `Future` and we could do a `map` instead of a `flatMap`?